### PR TITLE
Adding link to eosio training

### PR DESCRIPTION
**Pull request by bobgt**
As per requested by Bryant Neilson, adding link to EOSIO training for the Development Environment setup course.

### CODE REVIEW

File: docs/30_getting-started-guide/20_local-development-environment/index.md [hunk 1]
```diff
@@ -16,3 +16,8 @@ The following list provides instructions to prepare your local development envir
 3. [Create Development Wallet](30_development-wallet.md): Steps to create development wallet to store key pairs 
 4. [Start Keosd and Nodeos](40_start-nodeos-keosd.md): Steps to start `Keosd` and `Nodeos` 
 5. [Create Development Accounts](50_create-dev-accounts.md): Steps to create you first EOSIO blockchain accounts for smart contracts
+
+## EOSIO Training and Certification Center
+In addition to following the instructions given in the above sections to set up your local development environment, you can also enroll in EOSIO's online learning courses. The following online course provides instructions in a blended format ( textual and videos) to help you set up your local development environment:
```
Comment (iamveritas) [inline docs/30_getting-started-guide/20_local-development-environment/index.md:21]: can we avoid the "-ing"?

**Review by iamveritas — CHANGES_REQUESTED** [review]

File: docs/30_getting-started-guide/20_local-development-environment/index.md (hunk 1 above)
Comment (bobgt) [reply docs/30_getting-started-guide/20_local-development-environment/index.md:21]: You mean the `ing` In "addition to `following`"?

Comment (iamveritas) [reply docs/30_getting-started-guide/20_local-development-environment/index.md:21]: yes, it is the only gerund I can spot, and it sounds also weird... doesn't it?

**Review by lparisc — APPROVED** [review]
See comments in PR #401

File: docs/30_getting-started-guide/20_local-development-environment/index.md (hunk 1 above)
Comment (halsaphi) [reply docs/30_getting-started-guide/20_local-development-environment/index.md:21]: seems too wordy to me :

how about being imperative 

"Enroll in EOSIO's online learning courses for interactive guides which show you how to set up your local development environment:"

Comment (halsaphi) [reply docs/30_getting-started-guide/20_local-development-environment/index.md:21]: Actually thats bollux ...

Enroll in the EOSIO online learning courses and follow interactive guides which show you how to set up your local development environment:

or something like

**Review by halsaphi — APPROVED** [review]

File: docs/30_getting-started-guide/20_local-development-environment/index.md (hunk 1 above)
Comment (halsaphi) [reply docs/30_getting-started-guide/20_local-development-environment/index.md:21]: same comment for the other PR's
